components/prize_pool/wikis/brawlstars/prize_pool_custom.lua

```diff
@@ -0,0 +1,98 @@
+---
+-- @Liquipedia
+-- wiki=brawlstars
+-- page=Module:PrizePool/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Arguments = require('Module:Arguments')
+local Class = require('Module:Class')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local Namespace = require('Module:Namespace')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+local Variables = require('Module:Variables')
+
+local Opponent = require('Module:OpponentLibraries').Opponent
+
+local PrizePool = Lua.import('Module:PrizePool', {requireDevIfEnabled = true})
+
+local LpdbInjector = Lua.import('Module:Lpdb/Injector', {requireDevIfEnabled = true})
+local CustomLpdbInjector = Class.new(LpdbInjector)
+
+local CustomPrizePool = {}
+
+-- Brawlstars is still in the process of tier conversion
+-- hence "Monthly" is needed here until that is done
+local TIER_VALUE = {16, 8, 4, 2, Monthly = 2}
+
+local TYPE_MODIFIER = {offline = 1, ['offline/online'] = 0.75, ['online/offline'] = 0.75, default = 0.65}
+
+local HEADER_DATA = {}
+
+-- Template entry point
+function CustomPrizePool.run(frame)
+	local args = Arguments.getArgs(frame)
+
+	args.allGroupsUseWdl = true
+	args.groupScoreDelimiter = '-'
+
+	local prizePool = PrizePool(args):create()
+
+	HEADER_DATA.tournamentName = args['tournament name']
```

Comment:
Same

Reply:
it is functionality the old ppt has
basically it allows overwritng the value for the tournament key in lpdb
i think it mostly is used if there were >1 ppt on a page (at least that was the use cases we had on sc2 for it)

Reply:
https://liquipedia.net/brawlstars/index.php?search=insource%3A%22tournament+name%22&fulltext=1
Doesn't seem to be used. Probably just a copy pasta from some other wiki